### DISCUSSION TITLE
Edit submitted data from Admin interface

### DISCUSSION THREAD

**Discussion by PanickingLynx** [sulu/SuluFormBundle#335] [Q&A] [answered]
Hello sulu community,
I would like to edit data from a submitted form.
The form would contain a "Status" hidden field, which then could be edited from the admin view like "Submitted", "In-Progress", "Closed".
Is this possible in any way since I can't seem to find a button for editing form data.

Best regards
_Panic_

**Comment by alexander-schranz** [comment, marked as answer]
Currently nothing like this is implemented in the SuluFormBundle. I personally never had to usecase for "Contact Forms" to have such things. And for all other cases I have a custom Business Model and so create a "Custom Entity" and then I'm not using the SuluFormBundle. I know projects which did forward Submitted Data into Ticket Systems but this are currently implemented in project itself via FormSavePost event and not part of the bundle itself.

What kind of form are you building and what are you doing with the submitted data?

**Reply by PanickingLynx** [reply]
I am currently building a tenant report form. To report for example issues with flats and buildings out for rent. We would like to use SuluFormBundle for a sort of ticketing system I suppose yes.
